crossing-training.lic

```diff
@@ -684,7 +689,8 @@ class CrossingTraining
     if DRSkill.getxp('Athletics') < 29
       walk_to(2245)
       if UserVars.athletics < 550
-        practice_branch
+        @climbing_target['target'] = 'branch'
```

Comment:
If this does work, won't that mean that `@climbing_target` is defined? I think that would mean the next time Athletics is trained, the `train_athletics` method would take the first branch, but `@climbing_target['id']` would not be defined.